### DISCUSSION TITLE
Are there any copyright issues with using the generated content?

### DISCUSSION THREAD

**Discussion by deshetti** [Kovah/CorporateLorem#9] [Q&A]
Hi @Kovah 

Looks like an interesting project. We are working on an open-source CMS and developing a few free website themes for the same to demo the functionality of the CMS. I wanted to check if the content generated through this tool could be used for these demo themes that will be published online. Would we run into any copyright issues?

**Comment by Kovah** [comment]
Hi. Please consult a lawyer to get an answer to this specific question as I can't answer it.

**Reply by deshetti** [reply]
@Kovah Thanks for your quick response. This info that it might not be safe to use on public sites or at least that you have to consult an expert before doing so, should be in the README in bold IMO. I have created a [PR](https://github.com/Kovah/CorporateLorem/pull/10) to add that note.

The USP of this kind of generated text goes for a toss if folks use it in public applications like they do with Lorem Ipsum and get hit with a copyright notice.

It is a good project but, I might be missing some context on use cases if we can't publish it online. One thing you could do is to use royalty-free content like https://litipsum.com
